style.css

```diff
@@ -78,8 +83,182 @@ nav .menu-button {
   margin-left: 24px;
 }
 
+/* style social medial icons */
 .social-icons {
   display: flex;
   gap: 20px;
-  margin-top: 10px;
+  margin: 10px 0 0 10px;
+}
+
+/* position work section */
+.work-section {
+  position: relative;
+  top: 90%;
+}
+
+/* style work section part */
+.projects {
+  margin: 114px auto;
+  background-color: white;
+  height: 585px;
+  width: 327px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
```

Comment:
- [ ] As demanded by the project requirements. Kindly use grid to lay out your work section and use flexbox to lay out the contents of all your project cards.